.gitignore

```diff
@@ -3,12 +3,19 @@
 .bundle
 build/*
 tools/chef/cookbooks/*
+tools/chef/site-cookbooks/**/cookbooks/*
 tools/chef/tmp/*
 tools/assets/*
 bin/*
 .DS_Store
 secret.key
 
+# Chef
+**/.kitchen/
+**/cucumber/Gemfile.lock
+tools/chef/site-cookbooks/*/Berksfile.lock
+tools/chef/.chef/*_encrypted_data_bag_secret
```

Comment:
I guess at least having separate keys means the chef artifacts left on pipeline servers can't be used to decrypt production data bags